### PR TITLE
feat: Added GitHub Link in Footer

### LINKED ISSUE #79: [FEATURE] GitHub link to repo in footer
### Description

So people can easily log issues and star the repo add a GitHub link `https://github.com/EddieHubCommunity/CreatorsRegistry` to the footer

File to update `src/components/Footer.js`

Screenshot 2024-06-25 at 00 31 03

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by devesh-2002**
## Fixes Issue
closes #79

## Changes proposed

Added the GitHub Link in the Footer using Link tag. 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

image

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
-            All rights reserved | v{data.version}
-          </span>
+    <footer className="border-t border-gray-200 py-8">
+      <div className="mx-auto max-w-3xl px-4 sm:px-6 lg:max-w-7xl lg:px-8 flex justify-between items-center">
+        <div className="text-sm text-gray-500">
+          &copy; {new Date().getFullYear()} EddieHub All rights reserved | v{data.version}
         </div>
+        <Link href={githubUrl}>
+            {githubUrl}
+
```
Comment (eddiejaoude) [inline src/components/Footer.js:15]:
```suggestion
```

File: src/components/Footer.js [hunk 2]
```diff
@@ -1,17 +1,18 @@
-import data from "@/config/app.json";
+import data from '@/config/app.json';
```
Comment (eddiejaoude) [inline src/components/Footer.js:1]: This should remain as double quotes to be consistent with our project `"`, please check you are using our prettier settings

File: src/components/Footer.js [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
+import Link from 'next/link';
```
Comment (eddiejaoude) [inline src/components/Footer.js:2]: Here is the fix...

```suggestion
import data from "@/config/app.json";
import Link from "next/link";
```

File: src/components/Footer.js [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
-            All rights reserved | v{data.version}
-          </span>
+    <footer className="border-t border-gray-200 py-8">
+      <div className="mx-auto max-w-3xl px-4 sm:px-6 lg:max-w-7xl lg:px-8 flex justify-between items-center">
+        <div className="text-sm text-gray-500">
+          &copy; {new Date().getFullYear()} EddieHub All rights reserved | v{data.version}
         </div>
+        <Link href={githubUrl}>
+            {githubUrl}
+        </Link>
```
Comment (eddiejaoude) [inline src/components/Footer.js:15]:
```suggestion
        <Link href={githubUrl}>{githubUrl}</Link>
```

File: src/components/Footer.js [hunk 5]
```diff
@@ -1,17 +1,18 @@
...
-          <span className="block sm:inline">
-            &copy; {new Date().getFullYear()} EddieHub
-          </span>{" "}
-          <span className="block sm:inline">
-            All rights reserved | v{data.version}
-          </span>
+    <footer className="border-t border-gray-200 py-8">
+      <div className="mx-auto max-w-3xl px-4 sm:px-6 lg:max-w-7xl lg:px-8 flex justify-between items-center">
+        <div className="text-sm text-gray-500">
+          &copy; {new Date().getFullYear()} EddieHub All rights reserved | v{data.version}
```
Comment (eddiejaoude) [inline src/components/Footer.js:11]: Formatting here is incorret

```suggestion
          &copy; {new Date().getFullYear()} EddieHub All rights reserved | v
          {data.version}
```

File: src/components/Footer.js (hunk 2 above)
Comment (eddiejaoude) [reply src/components/Footer.js:1]: I will fix in another comment

**Review by eddiejaoude — APPROVED** [review]
Thank you for your contribution 👍 

I have made the formatting changes so we can get this merged 🎉

**Comment by devesh-2002** [comment]
Thanks a lot for all the help @eddiejaoude!

**Pull request merged**